src/status_im/ui/screens/wallet/subs.cljs

```diff
@@ -36,18 +36,6 @@
           str)
       "...")))
 
-(reg-sub :portfolio-change
```

Comment:
Can you explain why this is no longer needed?

Reply:
Portfolio changes are not displayed anymore, this was a leftover.